site/RoomDraw/src/app/login/login.component.html

```diff
@@ -0,0 +1,20 @@
+<div class = "body">
+    <img src="assets/bc-campus-blur-v2.jpg" class="bcCampusBackground" alt="Benedictine College" style="width:100%">
```

Comment:
Can you change `class` to `id`?  `id` makes more sense when we're only referring to a single object.  `class` is used more for multiple related objects that will have similar styles.

Reply:
Also, remove style attributes from html and put into css.

Reply:
One more nitpick: remove spaces around equal signs. E.g. `<div class="body">`